sunbird/common_config/config.ini

```diff
@@ -49,24 +56,75 @@ editsvgtemp = private/mlcore/api/v1/certificateTemplates/createSvg?baseTemplateI
 uploadcertificatetosvg = private/mlcore/api/v1/certificateTemplates/uploadTemplate/
 updatecertificatesolu = private/mlcore/api/v1/solutions/update/
 updateprojecttemplate = private/mlprojects/api/v1/project/templates/update/
+
+
+; projecttemplatesbulkcreate = /v1/project/templates/bulkCreate
+; projecttemplatesbulkupdate = /v1/project/templates/bulkUpdate
+; projecttemplatesupdate = /v1/project/templates/update/
+; projecttemplateslistbyids = /v1/project/templates/listByIds
+; projecttemplatesimportProjectTemplate = /v1/project/templates/importProjectTemplate/
+; projecttemplatesunnatidetails = /unnati/v1/project/templates/details/
+; projecttemplatesdetails = /v1/project/templates/details
+
+; solutioncreate = /v1/solutions/create
+; solutionupdate = /v1/solutions/update/{solutionId}
+; solutionaddRolesInScope = /v1/solutions/addRolesInScope/{solutionId}
+; solutiondetailsBasedOnRoleAndLocation = /v1/solutions/detailsBasedOnRoleAndLocation/{solutionId}
+; solutionremoveRolesInScope = /v1/solutions/removeRolesInScope/{solutionId}
+; solutionfetchLink = /v1/solutions/fetchLink/{solutionId}
+; solutionaddEntitiesInScope = /v1/solutions/addEntitiesInScope/{solutionId}
+; solutionremoveEntitiesInScope = /v1/solutions/removeEntitiesInScope/{solutionId}
+; solutionforUserRoleAndLocation = /v1/solutions/forUserRoleAndLocation
+; solutiontargetedSolutions = /v1/solutions/targetedSolutions
+; solutionverifyLink = /v1/solutions/verifyLink/{link}
+; solutionverifySolution = /v1/solutions/verifySolution/{solutionId}
+; solutionlist = /v1/solutions/list
+; solutiongetDetails = /v1/solutions/getDetails/{solutionId}
+
+; programscreate = /v1/programs/create
+; programsupdate = /v1/programs/update/{programId}
+; programsdetails = /v1/programs/details/{programId}
+; programsaddRolesInScope = /v1/programs/addRolesInScope/{programId}
+; programsaddEntitiesInScope = /v1/programs/addEntitiesInScope/{programId}
+; programsremoveRolesInScope = /v1/programs/removeRolesInScope/{programId}
+; programsremoveEntitiesInScope = /v1/programs/removeEntitiesInScope/{programId}
+; programslist = /v1/programs/list
+; programsjoin = /v1/programs/join/{programId}
+
+
 onelogo-onesignature = 1L1S
 onelogo-twosignature = 1L2S
 twologo-onesignature = 2L1S
 twologo-twosignature = 2L2S
-[production]
-host = # Add host Ip address followed by a slash /
-internal_kong_ip = # Add Internal kong Ip address followed by a slash /
-keyclockapibody = 
+[local]
+; host = http://localhost:3569/
+; host1 = http://localhost:5000/
+; internal_kong_ip = http://localhost:3569/
+; keyclockapibody = '{"email": "rahul@yahoo.com","password": "password"}'
+; email = rahul@yahoo.com
+; password = password
+; internal-access-token = 8c3a94f0931e01a4940a
+; keyclockapiurl = user/v1/account/login
+
+
+; DEV_testing
+host = https://dev.elevate-apis.shikshalokam.org/
+host1 = https://dev.elevate-apis.shikshalokam.org/
```

Comment:
what is host1? Can we not do it using a single variable?

Reply:
yes @adithyadinesh0412  i though you will test and check in your local so i kept all the data in congig.ini